### ISSUE TITLE
Changing a column from required to not required wipes all data in that column

### ISSUE THREAD

**Issue by jwld** [prisma/prisma2#1240]
Reproduce by creating a table with a required column, then changing it to optional while running prisma dev. It should be possible to do this without wiping all the data in that column.

**Comment by malekjaroslav** [comment]
Related https://github.com/prisma/lift/issues/267

**Comment by pantharshit00** [comment]
Yes, can you please test it out again with alpha `npm install -g prisma2@alpha`? I was also unable to reproduce this with alpha.

Might possibly a duplicate of prisma/lift#267 as reported by @malekjaroslav

**Comment by jwld** [comment]
I'm still seeing this behaviour for some reason. I installed the same alpha as you - `prisma2@2.0.0-alpha.487`.

Some other info in case it's relevant:
- OS is macOS Catalina 10.15.2
- Postgres version is 11.5
